apps/web/src/services/asset-inventory/collector/collector-main/collector-page-store.ts

```diff
@@ -64,5 +65,11 @@ export const useCollectorPageStore = defineStore('collector-page', {
                 throw e;
             }
         },
+        async setSelectedCollector(id) {
+            const itemIndex = this.collectors.findIndex(
+                (item) => item.collector_id === id,
+            );
+            this.selectedCollect = this.collectors[itemIndex];
+        },
```

Comment:
Question: Why was this action implemented as async? It doesn't seem to require asynchrony internally!